src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPluginAction.java

```diff
@@ -22,7 +22,7 @@ public Object getTarget() {
         return new EnvInjectVarList(Maps.transformEntries(envMap,
                 new Maps.EntryTransformer<String, String, String>() {
                     public String transformEntry(String key, String value) {
-                        return getSensibleVariables().contains(key) ? "********" : value;
+                        return getSensibleVariables() != null && getSensibleVariables().contains(key) ? "********" : value;
```

Comment:
In order to be 100%-sure about the value, call the method and assign its output to a variable before checking return values